libcaja-private/caja-file-conflict-dialog.c

```diff
@@ -619,11 +623,18 @@ caja_file_conflict_dialog_init (CajaFileConflictDialog *fcd)
     details->titles_vbox = widget;
 
     /* Setup the hboxes to pack file infos into */
+#if GTK_CHECK_VERSION (3, 0, 0)
+    vbox2 = gtk_vbox_new (FALSE, 12);
+    gtk_widget_set_halign (vbox2, GTK_ALIGN_START);
+    gtk_widget_set_margin_start (vbox2, 12);
+    gtk_box_pack_start (GTK_BOX (vbox), vbox2, FALSE, FALSE, 0);
+#else
     alignment = gtk_alignment_new (0.0, 0.0, 0.0, 0.0);
```

Comment:
Second argument means yalign is at start - should be probably set with valign in GTK+3?